src/main/java/org/springframework/integration/kafka/inbound/KafkaMessageDrivenChannelAdapter.java

```diff
@@ -55,11 +61,18 @@
  */
 public class KafkaMessageDrivenChannelAdapter<K, V> extends MessageProducerSupport implements OrderlyShutdownCapable {
 
+	/**
+	 * Header name/retry context variable for the raw received record (or records).
+	 */
+	public static final String RAW_RECORD = "record";
```

Comment:
This is bad name. Needs prefix. 
`kafka_record` ?

OTOH, can't this constant migrate to the Spring Kafka's `KafkaHeaders` ?